site/content/docs/4.3/customize/optimize.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Optimize
+description: Keep your projects lean, responsive, and maintainable so you can deliver the best experience and focus on more important jobs.
+group: customize
+toc: true
+---
+
+## Lean Sass imports
+
+When using Sass in your asset pipeline, make sure you optimize Bootstrap by only `@import`ing the components you need. Your largest optimizations will likely come from the `Layout & Components` section of our `bootstrap.scss`.
+
+{{< scss-docs name="import-stack" file="scss/bootstrap.scss" >}}
+
+
+If you're not using a component, comment it out or delete it entirely. For example, if you're not using the carousel, remove that import to save some file size in your compiled CSS. Keep in mind there are some dependencies across Sass imports that may make it more difficult to omit a file.
+
+## Lean JavaScript
+
+Bootstrap's JavaScript includes every component in our primary dist files (`bootstrap.js` and `bootstrap.min.js`), and even our primary dependency (Popper.js) with our bundle files (`bootstrap.bundle.js` and `bootstrap.bundle.min.js`). While you're customizing via Sass, be sure to remove related JavaScript.
+
+## Autoprefixer browserslist.rc
+
+Bootstrap depends on Autoprefixer to automatically add browser prefixes to certain CSS properties. Prefixes are dictated by our `browserslist.rc` file, found in the root of the Bootstrap repo. Customizing this list of browsers and recompiling the Sass will automatically remove some CSS from your compiled CSS, if there are vendor prefixes unique to that browser or version.
+
+## Use Purge CSS
+
+_Help wanted with this section, please consider opening a PR. Thanks!_
```

Comment:
A few things to consider with this:
- we'd have to maintain an exclusion list of some sort (PurgeCSS WhiteList) since it basically parses HTML and JS to determine which selectors can be applied or not, it's usually missing some (that could be dynamically computed in JS, or not in the initial DOM for some reason). This would need some tests and context, I guess.
- mention various extractors too, to prevent users from struggling with basic configuration.
- I think we can search for posts and references, this has probably been covered several times.

Reply:
Adding links and articles for now. This could be better suited in the starter npm project.